### PR TITLE
feat(Manager): design implementation for homepage

### PR DESCRIPTION
**Pull request by nikhitamore**
## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/src/views/HomePage/components/WebpropertiesTable/Tutorials.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+const tutorialsData = [
+  {
+    title: 'SPAship Manager & CLI Guide',
+    link: 'https://drive.google.com/file/d/150OyktZdmqMXKwNS1mDkqIo7ZvgyCCGp/view'
+  },
+  {
+    title: 'On-Boarding to SPAship Cloud Native Version',
+    link: 'https://source.redhat.com/groups/public/spaship/blog_article/onboarding_to_spaship_cloud_native_version'
+  }
```
Comment (soumyadip007) [inline packages/manager/src/views/HomePage/components/WebpropertiesTable/Tutorials.tsx:12]: @nikhitamore can we read it from the env variables

File: packages/manager/src/views/HomePage/components/WebpropertiesTable/VideoGuidesCard.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    },
+    {
+      title: 'Containerized deployments complete workflow',
+      link: 'https://drive.google.com/file/d/1g-obdU_RT1kQOgDKM150s_eVyiYUElKa/view?usp=sharing'
+    },
+    {
+      title: 'SPAship CLI Instruction',
+      link: 'https://drive.google.com/file/d/1G4B8AKu_8M6fVEMd8WtXHgLhf__Xl2mt/view?usp=sharing'
+    }
+  ];
```
Comment (soumyadip007) [inline packages/manager/src/views/HomePage/components/WebpropertiesTable/VideoGuidesCard.tsx:18]: read these from the env variables or use the `documentation` api

**Review by soumyadip007 — CHANGES_REQUESTED** [review]

File: packages/manager/src/views/HomePage/components/WebpropertiesTable/VideoGuidesCard.tsx (hunk 2 above)
Comment (nikhitamore) [reply packages/manager/src/views/HomePage/components/WebpropertiesTable/VideoGuidesCard.tsx]: Done,

File: packages/manager/src/views/HomePage/components/WebpropertiesTable/Tutorials.tsx (hunk 1 above)
Comment (nikhitamore) [reply packages/manager/src/views/HomePage/components/WebpropertiesTable/Tutorials.tsx]: done

**Review by soumyadip007 — APPROVED** [review]
LGTM

**Pull request merged**